web/src/main/java/com/epam/rft/atsy/web/controllers/rest/CandidateController.java

```diff
@@ -88,4 +103,10 @@ private SearchOptions parseFilters(String filterJSON) {
       return EMPTY_SERCH_OPTIONS;
     }
   }
+
+  @Override
+  public ResponseEntity<RestResponse> saveOrUpdate(@Valid @RequestBody CandidateDTO dto,
+                                                   BindingResult bindingResult, Locale locale) {
+    return null;
```

Comment:
If this method should not do anything, you could return an exception if it is called.